README.md

```diff
@@ -120,3 +120,10 @@ This is only necessary for developing for the ESP32 sensor processor board, and
 2. Follow the instructions for how to install the framework for whichever platform you're developing on.
 3. Once the JSON-RPC libraries are installed, the sensor code will be able to be compiled
 
+### Set up JSON-RPC for Django 
```

Comment:
You can set up django-json-rpc more simply by using pip: `pip3 install django-json-rpc`